src/components/textarea/textarea.stories.mdx

```diff
@@ -159,6 +159,10 @@ You can use the `required` prop to indicate if the field is mandatory.
 
 #### Customer Border Radius
 
+The `borderRadius` prop accepts a single string that will applied to each corner or an array of values that will be appied in the same order as the CSS border-radius values. 
```

Comment:
**suggestion** (non-blocking): I think these docs could be a bit clearer imo - because 2 or 3 values are also valid, not just an array of 4. This also doesn't mention that the values must all be design tokens.

Maybe something like this?

```
The `borderRadius` prop accepts either a single design token, or an array of them, that will be applied as the CSS `border-radius` property. See MDN docs (add link here) for examples of how this works with multiple values.
```